modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/ps/PostgisPreserveTopologyOnlineTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.postgis.ps;
+
+import org.geotools.data.postgis.PostgisPreserveTopologyTestSetup;
+
+public class PostgisPreserveTopologyOnlineTest
```

Comment:
I must be missing something, what is the difference with PostgisDontPreserveTopologyOnlineTest above? Both classes setup:
```
new PostgisPreserveTopologyTestSetup(new PostGISPSTestSetup());
```

Reply:
Agreed. @mbarto  multiple tests can share the same test setup, if the actions that need to be done to prepare the database are the same.

Reply:
@aaime @jodygarnett I made two classes to have a different createDataStoreFactoryParams, so that I can set the flag on for one class and off in the other one and have two different stores. Can you think of a different (better) way of doing that? I couldn't find one. Maybe I can make one extend the other one and just override createDataStoreFactoryParams?

Reply:
Nah, it's ok, I just forgot that every test in PostGIS is run with and without prepared statements, the classes in the ``ps`` sub-package subclass the one in the upper level package.